src/shared/utils/urlGenerator.ts

```diff
@@ -133,8 +131,8 @@ export const checkCurrentActivityPage = (url: string) => ({
 
 export const checkCurrentActivityFlowPage = (url: string) => ({
   isAbout: new RegExp(`${getAppletActivityFlowPageRegexp(Path.About)}`).test(url),
-  isBuilder: new RegExp(`${getAppletActivityFlowPageRegexp(Path.FlowBuilder)}`).test(url),
-  isSettings: new RegExp(`${getAppletActivityFlowPageRegexp(Path.FlowSettings)}`).test(url),
+  isBuilder: new RegExp(`${getAppletActivityFlowPageRegexp(Path.Builder)}`).test(url),
```

Comment:
should not FlowBuilder|FlowSettings be removed from enum as well?

Reply:
I removed it